catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -307,6 +312,7 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
+                        <!--todo: why do we embed spark-core if it's a bundle?-->
```

Comment:
❓ Without a ticket number associated with this, it needs to be taken care of on this PR. Should we create a ticket for investigating this?